# Define the goal
geometry_msgs/PointStamped[] centroids
string[] colors
geometry_msgs/PointStamped red_goal
geometry_msgs/PointStamped blue_goal
---
# Define the result
bool result
geometry_msgs/PointStamped obj_centroid
---
# Define a feedback message